### ISSUE TITLE
Error from Uncaught application error

### ISSUE THREAD

**Issue by wgottschalk** [samccone/bundle-buddy#44]
Not sure what's happening here. I just cloned the repo, ran yarn build, then npx serve -s build

`Uncaught application error`:
```Cannot read property 'key' of undefined
----
TypeError: Cannot read property 'key' of undefined
    at G.render (http://localhost:5000/static/js/5.ec9119e6.chunk.js:1:23789)
    at za (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:162000)
    at La (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:161793)
    at mu (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:197440)
    at cl (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:188919)
    at ul (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:188844)
    at Zu (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:185853)
    at http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:137238
    at t.unstable_runWithPriority (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:212818)
    at Wo (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:136947)
    at Qo (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:137183)
    at A (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:211756)
    at MessagePort.x.port1.onmessage (http://localhost:5000/static/js/2.d75fb8b9.chunk.js:2:210512)```

**Comment by wgottschalk** [comment]
Also I'm on the git branch `bundle-explorer` and when I run `yarn start` I get the following error:
```
Uncaught TypeError: Cannot read property 'key' of undefined
    at Home.render (Home.tsx:21)
    at finishClassComponent (react-dom.development.js:17160)
    at updateClassComponent (react-dom.development.js:17110)
    at mountLazyComponent (react-dom.development.js:17358)
    at beginWork (react-dom.development.js:18602)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at beginWork$1 (react-dom.development.js:23203)
    at performUnitOfWork (react-dom.development.js:22154)
    at workLoopSync (react-dom.development.js:22130)
    at performSyncWorkOnRoot (react-dom.development.js:21756)
    at react-dom.development.js:11089
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11084)
    at workLoop (scheduler.development.js:597)
    at flushWork (scheduler.development.js:552)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:164)
```

**Comment by wgottschalk** [comment]
Finally, a screenshot of the code 
![image](https://user-images.githubusercontent.com/11728959/84582409-76ef7a80-ada0-11ea-934b-422ed1ec4000.png)

Let me know if I'm missing anything here

**Comment by samccone** [comment]
@wgottschalk oh no :/ looking into this now!

**Comment by samccone** [comment]
fixed in 0a0dae3355a4d1bac93f8b9e4fabfa652ad96445
